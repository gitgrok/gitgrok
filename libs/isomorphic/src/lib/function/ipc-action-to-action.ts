import { IAction } from "../interfaces/action.interface";
import { IIpcAction } from "../interfaces/ipc-action.interface";

export function ipcActionToAction({ actionType, actionProps }: IIpcAction): IAction {
    return { ...actionProps, type: actionType };
};
