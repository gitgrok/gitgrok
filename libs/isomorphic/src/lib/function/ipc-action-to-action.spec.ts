import { ipcActionToAction } from './ipc-action-to-action';
const actionType = 'actionType';
const actionProps = {any: 'thing'}

describe('ipcActionToAction', () => {
    it('worx', () => {
        expect(ipcActionToAction({actionType, actionProps})).toMatchSnapshot();
    });
});