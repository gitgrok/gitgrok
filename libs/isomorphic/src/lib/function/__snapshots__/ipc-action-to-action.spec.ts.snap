// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ipcActionToAction worx 1`] = `
Object {
  "any": "thing",
  "type": "actionType",
}
`;
